NitroxServer/Server.cs

```diff
@@ -177,10 +180,10 @@ public bool Start(CancellationTokenSource cancellationToken)
                 {
                     Log.Info("Starting to load all batches up front.");
                     Log.Info("This can take up to several minutes and you can't join until it's completed.");
-                    Log.Info($"{entityManager.GetAllEntities().Count} entities already cached");
-                    if (entityManager.GetAllEntities().Count < 504732)
+                    Log.Info($"{entityRegistry.GetAllEntities().Count} entities already cached");
+                    if (entityRegistry.GetAllEntities().Count < 504732)
```

Comment:
Is this magic number from MrPurple still correct ?

Reply:
Probably not but we currently can't reliable execute the server with `CreateFullEntityCache=True` so we need to update this number later.